src/import-utils.ts

```diff
@@ -94,6 +117,7 @@ export async function inlineHtmlImport(
   }
 
   const importDoc = parse5.parseFragment(importSource);
+  debaseDocument(resolvedImportUrl, importDoc);
```

Comment:
Thinking about it further, since the purpose of `rewriteImportedUrls` is to rewrite urls in a document because the base they were build on has shifted, the term "Imported" is misleading, since we actually have two contexts where we manipulate the base of a document:

1. We encounter a base tag and we have to remove it and emulate it because we're merging documents together.
2. We inline a document and have to modify the contents of it to be as if it has a new base url.

So, for clarity, I've renamed a lot of things:
```
export function debaseDocument -> rewriteDocumentToEmulateBaseTag
export function rewriteImportedRelPath -> rewriteHrefBaseUrl
export function rewriteImportedUrls -> rewriteDocumentBaseUrl
function _rewriteImportedElementAttrUrls ->  _rewriteElementAttrsBaseUrl
function _rewriteImportedStyleTextUrls -> _rewriteCssTextBaseUrl
function _rewriteImportedStyleUrls -> _rewriteStyleTagsBaseUrl
function _setImportedDomModuleAssetpaths -> _setDomModuleAssetpaths
```

Also the parameters to all the functions have been renamed:
```
importDoc -> ast
importDocUrl -> oldBaseUrl
mainDocUrl -> newBaseUrl
```

I've also normalized the argument order of the functions so that the target (text or ast node) comes first, followed by oldBaseUrl and newBaseUrl.